tests/nftOrders.test.ts

```diff
@@ -807,7 +811,9 @@ describe('NFT Orders', () => {
         .toString(10)
     );
     expect(
-      new BigNumber(takerTokenERC20AfterBalance.toString()).toString(10)
+      new BigNumber(takerTokenERC20AfterBalance.toString())
+        .minus(augustusTakerTokenBalanceBeforeSwap.toString()) // if augustus contained some dust, it'll be transferred to the taker in the result of a swap
```

Comment:
Comment is wrong here. In NFT case, taker takes NFT, augustus balance goes to maker, taker just gets to keep some of his balance